From e8b9d8d856705bee0bd4a3957e691ef79f4fec66 Mon Sep 17 00:00:00 2001
From: Calle Laakkonen <calle.laakkonen@sofokus.com>
Date: Thu, 26 Sep 2024 15:19:06 +0300
Subject: [PATCH 02/15] DF: Enable sending with mailgun EU endpoint

---
 server/src/email/senders.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/server/src/email/senders.ts b/server/src/email/senders.ts
index 8dde04e3..2d23ea9b 100644
--- a/server/src/email/senders.ts
+++ b/server/src/email/senders.ts
@@ -58,6 +58,7 @@ function getMailOptions(transportType: any) {
           api_key: Config.mailgunApiKey || "unset-value",
           domain: Config.mailgunDomain || "unset-value",
         },
+        host: process.env.MAILGUN_ENDPOINT || "unset-value", // e.g. for EU region (DF customization)
       };
       return mg(mailgunAuth);
     case "aws-ses":
-- 
2.45.1

