From 6897e75d293d2321c257ae5a861d50ced9629af7 Mon Sep 17 00:00:00 2001
From: Calle Laakkonen <calle.laakkonen@sofokus.com>
Date: Thu, 26 Sep 2024 15:37:58 +0300
Subject: [PATCH 09/13] DF: Draw only isSelf icon

---
 client-participation/vis2/components/graphParticipants.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/client-participation/vis2/components/graphParticipants.js b/client-participation/vis2/components/graphParticipants.js
index 7eb36895..3eb36905 100644
--- a/client-participation/vis2/components/graphParticipants.js
+++ b/client-participation/vis2/components/graphParticipants.js
@@ -4,6 +4,7 @@ import * as globals from "./globals";
 import {VictoryAnimation} from "victory-core";
 
 const Participant = ({ptpt, tweenX, tweenY}) => {
+  if (!ptpt.isSelf) {return null};
   let picSize = ptpt.picture_size;
   return (
     <g
-- 
2.47.1

